modules/hcp-dr-oadp-restore.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="hcp-dr-oadp-restore_{context}"]
-= Restoring a hosted cluster by using {oadp-short}
+= Restoring a hosted cluster into the same management cluster by using {oadp-short}
```

Comment:
Later in this document we mention the step "Scale the `NodePool` resource to the desired number of replicas by running the following command": https://github.com/openshift/openshift-docs/pull/95105/files#diff-3b53b82c1264e7b22c813934ffd7bb532160440fb869bede37db8f441d41fb40R144
But the backup process above didn't have the step of scaling down the NodePool. 
I'm not completely sure whether it should be included in the backup process or not. The upstream docs have it for OADP 1.4 , not for OADP 1.5
Is this documentation for OADP 1.4 or 1.5 ?
@jparrill Could you confirm whether scaling down the NodePool is required and whether the step of scaling up again must match this "scale down" step in the backup process? Thanks

Reply:
I think this was leftover from a prior backup procedure where we did scale down the nodepool. I don't see that step in the published docs [1] anymore. Let me check if this is still a required backup step when restoring to the same management cluster.

Reply:
Ok I think scaling up the nodepool is no longer needed. It was removed from the backup process in OCP 4.17 [1], but it looks like the restore process hadn't remove this step at this time. Let's remove the scale up step in the restore document.

[1] draft document: https://docs.google.com/document/d/1c6Cx4tq09PcxYPIwcNGEUx4O8NPgvBuKgc7bGfZSQNg/edit?tab=t.0

Reply:
Thank you!